lambda_powertools/prometheus.py

```diff
@@ -0,0 +1,78 @@
+import os
+import json
+import sys
+
+modulename = 'prometheus_client'
+if modulename in sys.modules:
+    raise Exception("prometheus_client already imported, lambda_powertools.prometheus must be imported before")
+
+os.environ['PROMETHEUS_DISABLE_CREATED_SERIES'] = 'True'
+import prometheus_client
+from prometheus_client import CollectorRegistry
+
+prometheus_client.REGISTRY.unregister(prometheus_client.GC_COLLECTOR)
+prometheus_client.REGISTRY.unregister(prometheus_client.PLATFORM_COLLECTOR)
+prometheus_client.REGISTRY.unregister(prometheus_client.PROCESS_COLLECTOR)
+
+
+def reset():
+    for collector in list(prometheus_client.REGISTRY._collector_to_names.keys()):
```

Comment:
You don't need to create a list to iterate over the keys. You could simply use:
`for collector in prometheus_client.REGISTRY._collector_to_names.keys():` 
Or simply do:
`for collector in prometheus_client.REGISTRY._collector_to_names:`